### PR TITLE
No blank cvpage test

### PR DESCRIPTION
**Pull request by sambible**
Test for a strange BZ we received, where certain characters in other languages ( Like French, in this example ) broke the CV page, and caused it to appear blank. 

This test simply changes the language for a user to French, and navigates to the CV page and asserts it's not blank. 

This should **not** be used as an example of increasing or beginning automation efforts around translations, as there are other teams in red hat that do this. This is is simply automating something that got through as a BZ, and is here mostly as a guard against regressions of that particular BZ. 

Requires sister airgun PR: https://github.com/SatelliteQE/airgun/pull/991

### CODE REVIEW

**Comment by lhellebr** [comment]
I reviewed the airgun part and I'm ready to review this, may I just suggest making this in to exactly two commits, one with the file move and renaming new to old, and the other with new content? So the reviewers don't have a hard time identifying what is what.

**Comment by sambible** [comment]
There's an older PR that should get merged soon, that has the move and rename as the majority of the content.

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -40,3 +40,34 @@ def test_positive_create_cv(session, target_sat):
...
+    user_password = gen_string('alpha')
+    user = target_sat.api.User(
+        default_organization=module_org,
+        organization=[module_org],
+        password=user_password,
+        admin=True,
+    ).create()
+    with target_sat.ui_session(user=user.login, password=user_password) as session:
+        session.user.update(user.login, {'user.language': 'Français'})
+        assert session.contentview_new.check_if_blank_in_french()
```
Comment (shubhamsg199) [inline tests/foreman/ui/test_contentview.py:73]: Looking at `check_if_blank_in_french`, I don't think that asserts anything.
Can you add a different assertion to check?

**Review by shubhamsg199 — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_contentview.py (hunk 1 above)
Comment (sambible) [reply tests/foreman/ui/test_contentview.py:73]: It's simply asserting that the table exists - the bug in this case was that in different languages, i.e. french, the page wouldn't display at all. So in this case, simply asserting that the table exists is sufficient.

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
igger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_contentview.py -k 'test_no_blank_page_on_language_switch'
airgun: 991

**Comment by sambible** [comment]
PRT passed pre merge stuff, going to rebase this one locally.

File: tests/foreman/ui/test_contentview.py [hunk 2]
```diff
@@ -40,3 +40,36 @@ def test_positive_create_cv(session, target_sat):
...
+
+@pytest.mark.tier2
+def test_no_blank_page_on_language_switch(session, target_sat, module_org):
+    """Able to view the new CV UI when the language is set to something other
+    than English
+
+    :id: d8745aca-b199-4c7e-a970-b1f0f5c5d56c
+
+    :steps:
+        1. Change the Satellite system language to french or czech
```
Comment (ColeHiggins2) [inline tests/foreman/ui/test_contentview.py:53]: Looks like we only update the language to french. lets remove "or czech"

**Review by ColeHiggins2 — APPROVED** [review]

File: tests/foreman/ui/test_contentview.py (hunk 2 above)
Comment (ColeHiggins2) [reply tests/foreman/ui/test_contentview.py:53]: comments not blocking

**Review by Griffin-Sullivan — APPROVED** [review]

**Pull request merged**